src/test/fixtures/fixtures_test.ts

```diff
@@ -135,6 +135,12 @@ suite('Fixtures', () => {
           '--add-import-path',
         ].concat(fixtureTestConfig.options || []));
 
+        if (process.env.UPDATE_EXPECTED === 'true') {
```

Comment:
It feels weird to run a tool through an existing test, that actually generates its own fixture, and then become a noop test. If the purpose is to DRY up our fixture generation logic, could we instead move this logic into a shared place that both generation tool & tests could pull from?

Reply:
Ok, I added a new library and depend on it in both cases, PTAL.

I've used the "record mode" pattern I had before -- I kind of like it because it's so simple, it makes it harder to accidentally get your golden generation out of sync from the test itself, and you can use mocha's `--grep` flag to easily control which test golden you are re-generating. But I concede that it is hacky.